docs/site/migration/models/mixins.md

```diff
@@ -566,30 +505,30 @@ import {Model} from '@loopback/repository';
 import {FindByTitleInterface} from './findByTitleInterface';
 import {param, get, getModelSchemaRef} from '@loopback/rest';
 
-export interface FindByTitleControllerMixinParms {
+export interface FindByTitleControllerMixinOptions {
   basePath: string;
-  modelClass: Constructor<{}>;
+  modelClass: Constructor<object>;
```

Comment:
As mentioned in https://github.com/strongloop/loopback-next/pull/4723#discussion_r383925889, I think this should be:

```suggestion
  modelClass: typeof Model;
```